### DISCUSSION TITLE
Equivalent for xterm's "altSendsEscape: False"

### DISCUSSION THREAD

**Discussion by gnarendran** [kovidgoyal/kitty#5421] [Q&A] [answered]
For alt+a, kitty generates "^[a" or "\x1b\x61". Is it possible to generate "\xe1", ie. instead of escape to set the 8th bit, by config, like what xterm does when Xresource altSendsEscape is False?

From a quick reading of the kitty documents could not find any way other than via mappings such as:
`map alt+a send_text normal,application \xe1`

Regards

**Comment by kovidgoyal** [comment, marked as answer]
No there isnt.

**Comment by gnarendran** [comment]
For anyone interested, attached the complete mappings required (standard keyboard) to emulate xterm's "altSendsEscape:False".
[emulationOfXtermAltSendsEscapeFalse.conf.txt](https://github.com/kovidgoyal/kitty/files/9432903/emulationOfXtermAltSendsEscapeFalse.conf.txt)
